server.js

```diff
@@ -2,7 +2,7 @@ const express = require('express')
 const app = express()
 
 app.get('/ping',(req,res)=>{
-    res.send('Ping')
+    res.json({message:'Pong'})
 })
 
 app.listen(3000,()=>{
```

Comment:
It's a good practice to handle errors in your express app. Consider adding an error handling middleware at the end of your middleware function stack. [important]